gbpservice/nfp/configurator/lib/schema.py

```diff
@@ -81,24 +81,10 @@
               'stitching_cidr': "",
               'stitching_interface_index': "",
               'provider_mac': "",
-              'stitching_mac': "",
```

Comment:
Yes..Unit tests need to be changed. That shall be done as part of the review comments addressal branch which shall incorporate these changes. Is that fine?